MetadataGenerator/AppDelegate.m

```diff
@@ -9,340 +9,342 @@
 #import "AppDelegate.h"
 @import libPhoneNumberiOS;
 
-
 @interface AppDelegate ()
 @end
 
-
 @implementation AppDelegate
 
-- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
-{
-    self.window = [[UIWindow alloc] initWithFrame:[[UIScreen mainScreen] bounds]];
-    self.window.backgroundColor = [UIColor blackColor];
-    [self.window makeKeyAndVisible];
+- (BOOL)application:(UIApplication *)application
+    didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+  self.window = [[UIWindow alloc] initWithFrame:[[UIScreen mainScreen] bounds]];
+  self.window.backgroundColor = [UIColor blackColor];
+  [self.window makeKeyAndVisible];
 
-    [self.window setRootViewController:[[UIViewController alloc] init]];
```

Comment:
Going to only 1 space seems  little rough.  Any thoughts to keep it at "2" or going with "4"?

Reply:
This is 2 spaces

Reply:
I'm a little confused.. we are at two...

Reply:
Sorry. Looked too quickly.  The block looked like it represented 1 space, instead of 2.  So...I guess I would be advocating for leaving it at 4 spaces, then (monday mornings... :frowning: )

![Case of the Mondays](http://i0.kym-cdn.com/entries/icons/original/000/018/456/case-of-the-mondays-careers.png)